data-root:
  files:
    var/log/neutron/neutron-server.log: |
      2022-07-04 20:03:32.405 2050177 ERROR ovsdbapp.backend.ovs_idl.connection ValueError: non-zero flags not allowed in calls to send() on <class 'eventlet.green.ssl.GreenSSLSocket'>
    sos_commands/dpkg/dpkg_-l: |
      ii  python3-openvswitch 2.17.0 amd64
  copy-from-original:
    - sos_commands/date/date
    - uptime
raised-bugs:
  https://bugs.launchpad.net/bugs/1980211: >-
    Installed package 'python3-openvswitch' with version
    2.17.0 has a known bug whereby if connections to the ovn
    southbound db are closed, the client fails to reconnect.
    This is usually resolved with a service restart and a
    fix is available as of openvswitch version 2.17.2.
